workon

```diff
@@ -0,0 +1,34 @@
+[1mdiff --git a/create_box/main.py b/create_box/main.py[m
+[1mindex 7cb7bc7..84746d0 100644[m
+[1m--- a/create_box/main.py[m
+[1m+++ b/create_box/main.py[m
+[36m@@ -1,16 +1,25 @@[m
+ """This is the entry point of the program."""[m
+ [m
+ [m
+[31m-def create_box(height, width, character):[m
+[32m+[m[32mdef create_box(height, width, character, border_only = False):[m
+     x=''[m
+     for i in range(0,height):[m
+[31m-        for j in range(0,width):[m
+[31m-            x+=character[m
+[32m+[m[32m        if i ==0 or i==height-1:[m
+[32m+[m[32m                x+=character*width[m
+[32m+[m[32m                x+='\n'[m
+         else:[m
+[31m-            x+='\n'[m
+[32m+[m
+[32m+[m[32m            for j in range(0,width):[m
+[32m+[m[32m                if border_only and j>=1 and j!=width-1:[m
+[32m+[m[32m                    x+=' '[m
+[32m+[m[32m                else:[m
+[32m+[m[32m                    x+=character[m
+[32m+[m[32m            else:[m
+[32m+[m[32m                x+='\n'[m
```

Comment:
I'm not sure. I was having some trouble pushing to git at one point. And when I push to git I just use $ git add .

The final code works ok.